### PR TITLE
add new date format: mmm d, yyyy

### PR DESCRIPTION
**Pull request by DIFSRIP**
solves: https://github.tools.sap/sProcurement/qmate/issues/318

### CODE REVIEW

File: src/reuse/modules/util/formatter.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ export class Formatter {
       hour = `0${hour}`;
     }
 
-    if (dd < 10) {
+    if (dd < 10 && format !== DateFormats.MONTH_DAY_YEAR_COMMA_SHORT) {
```
Comment (heraldantony) [inline src/reuse/modules/util/formatter.ts:165]: Should lowercase format before this check, the test fails otherwise.

Comment (DIFSRIP) [reply src/reuse/modules/util/formatter.ts:165]: right, will do that

**Review by heraldantony — DISMISSED** [review]
looks good

**Review by heraldantony — APPROVED** [review]
looks good

**Pull request merged**